test/0135/operation-id.test.js

```diff
@@ -0,0 +1,62 @@
+const { linterForAepRule } = require('../utils');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0135', 'aep-135-operation-id');
+  return linter;
+});
+
+test('aep-135-operation-id should find errors', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      '/test1/{id}': {
+        delete: {
+          responses: {
+            200: {
+              description: 'Ok',
+            },
+          },
+        },
+      },
+      '/test2/{id}': {
+        delete: {
+          operationId: 'random',
+        },
+      },
+    },
+  };
+  return linter.run(oasDoc).then((results) => {
+    expect(results.length).toBe(2);
+    const paths = results.map(({ path }) => path.join('.'));
+    expect(paths).toContain('paths./test1/{id}.delete');
+    expect(paths).toContain('paths./test2/{id}.delete.operationId');
+  });
+});
+
+test('aep-135-operation-id should find no errors', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
```

Comment:
separate Q: should the examples be using 3.0.3? I feel like we should try to adhere to 3.1 to stay up to date with the OAS spec.

Reply:
Good call. I opened an issue to track this.